script.js

```diff
@@ -1,4 +1,125 @@
+// There are 2 players and players take turns.
+// When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+// The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+// After both players have rolled and chosen dice order, the player with the higher combined number wins.
+//--------
+
+//Declare global variables and game mode (2 - one to roll and one to choose order)
+var gameMode = "roll dice"; // initial game mode
+var gameMode_rollDice = "roll dice"; //assign var to different game mode
+var gameMode_order = "pick dice order";
+var gameMode_reset = "reset";
+var diceNumber1 = "";
+var diceNumber2 = "";
+var numberList = [];
+var counter = 0;
+
+// main function
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  //Loop will cut after 2 rounds. Can change number as more players join.
+  if (counter < 2) {
+    //---When gameMode = "roll dice"
+    if (gameMode == gameMode_rollDice) {
+      //Input validation
+      if (input !== "") {
+        console.log("This is test");
+        return "click submit to start";
```

Comment:
Good practice of "returning early" instead of using myOutputValue and reassigning it throughout the code